src/Cli/dotnet/CommonOptions.cs

```diff
@@ -51,6 +51,15 @@ internal static class CommonOptions
             }.ForwardAsSingle(o => $"-property:TargetFramework={o}")
             .AddCompletions(Complete.TargetFrameworksFromProjectFile);
 
+        public static Option<string> ArtifactsPathOption =
+            new ForwardedOption<string>(
+                //  --artifacts-path is pretty verbose, should we use --artifacts instead (or possibly support both)?
```

Comment:
I like supporting both. We should ideally mention this in the doc's options for each command, such as in https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-build

Reply:
Wouldn't artifacts imply the produced output instead of their location?

Reply:
That's true. Nvm on the --artifacts path, still think we should include this in the docs tho!